Sources/Rendering/Core/AbstractMapper/index.js

```diff
@@ -24,9 +29,10 @@ function vtkAbstractMapper(publicAPI, model) {
 
   publicAPI.removeClippingPlane = (i) => {
     if (i < 0 || i >= 6) {
```

Comment:
shouldn't you compare with `model.clippingPlanes.length` instead of 6 ?

Reply:
That was from the original code base. But I think you are right we should make sure we are in array bounds.

Reply:
`setClippingPlanes` below also has a constraint only allowing six clipping planes. I will change this so we check array bounds but also still cap at six, unless we know why that old constraint no longer should apply.